addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -46,9 +46,10 @@ def _domain_project_id(self):
         return domain
 
     def _domain_employee_id(self):
+        domain = [('company_id', 'in', self._context.get('allowed_company_ids'))]
         if not self.user_has_groups('hr_timesheet.group_hr_timesheet_approver'):
-            return [('user_id', '=', self.env.user.id)]
-        return []
+            expression.AND([domain, ('user_id', '=', self.env.user.id)])
```

Comment:
If you select two companies, wouldn't you be able to see two similar employees like in the screenshot of the pad? The code seems ok, but you'll have to see with Alexandra if that point is ok with her :)

Reply:
@dise-odoo In that case, it is expected to have 2 employees for the same user, no? 🙂